plugins/org/src/components/Cards/OwnershipCard/useGetEntities.ts

```diff
@@ -130,6 +130,7 @@ export function useGetEntities(
   relationsType: string,
   isGroup: boolean,
   entityFilterKind?: string[],
+  entityLimit = Infinity,
```

Comment:
I wonder if we should also default this to the original limit, which would be 6 instead of `Infinity` just so that we don't break existing behaviour?

Reply:
@benjdlambert , Even if we pass _INFINITY_, it wont break the flow, it will just render all boxes (entities) into the DOM.
But if we restrict it's values to 6, then in that case, it will only show 6 boxes, even if we want to see all boxes on UI.
In that case, we always need to pass the value for entityLimit, if we want to see more than 6 boxes on UI.

So I think, keeping it as _INFINITY_ is fine, it's my thought. Also issue itself mentions all boxes should be visible
<img width="933" alt="image" src="https://user-images.githubusercontent.com/12490835/230055884-c34d5c6c-c5c5-4772-94eb-4ae22be4cc35.png">

Reply:
I'm not saying that it will break the flow, I'm saying it could be unexpected behaviour for the majority of users. Happy to keep the option in here, but this behaviour could be classed as opt-in until it becomes a pattern that people use a lot and we're happy to enable it by default.

Reply:
Got it. Will keep to 6. Thanks!